include/common_structs.h

```diff
@@ -1426,6 +1426,17 @@ typedef struct CollisionStatus {
     /* 0x1C */ Vec3f bombetteExplosionPos;
 } CollisionStatus; // size = 0x28
 
+typedef struct DecorationUnk {
+               s16 unk00;
```

Comment:
Offset comments and the size comment would be good (like we have for all our other structs).